### ISSUE TITLE
Error setting up Service Participant

### ISSUE THREAD

**Issue by ClaytonCalabrese** [jmmorato/openddsharp#67]
Hello, I was trying to use OpenDDSharp and was running into an error setting up the Service Participant.

I am on Windows 10, using Visual Studio 2019, and the most recent release of OpenDDSharp and the associated IDL extension.

I was following the instructions of the Hello World example.

The projects are able to compile but then run into this error upon trying to run.

```
(5456|16044) ERROR: Service_Participant::load_configuration import_config () returned -3
(5456|16044) ERROR: Service_Participant::get_domain_participant_factory: load_configuration() failed.

```

**Comment by jmmorato** [comment]
Hi,

Your `rtps.ini` config file should be corrupted. 

I would say you copied from another OS (Mac?) and the line ending is not supported on Windows. Try to create it from scratch in Windows and try again. 

Let me know if that work for you.

**Comment by ClaytonCalabrese** [comment]
I was able to get it to work.

I did not see an option to create an .ini file in visual studio so I was creating a .txt file, and then renaming it to an .ini.  Despite it appearing to change the file type in visual studio, it must have been doing it incorrectly.

By creating a .txt file in the file explorer then using notepad's save as to rename it into a .ini file, I was able to get OpenDDSharp to run.

Thank you for the help.
